react/components/molecules/tabs-text/tabs-text.js

```diff
@@ -45,13 +48,21 @@ export class TabsText extends PureComponent {
             animatedBarOffset: undefined
         };
         this.tabLayouts = {};
+        this.scroll = 0;
+        this.overflows = false;
     }
 
     onTabPress = tabSelectedIndex => {
         this.setState({ tabSelected: tabSelectedIndex }, () => {
             this._updateBar();
             this.props.onTabChange(this.state.tabSelected);
         });
+        this._scrollToTabIfOverflows(tabSelectedIndex);
+    };
+
+    onScroll = event => {
+        const XScroll = event.nativeEvent.contentOffset.x;
+        this.XScroll = XScroll;

```

Comment:
Our standard: variables must always start with lowercase.
```suggestion
        this.scroll = scroll;
```